PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewEmulatorForm.java

```diff
@@ -285,6 +286,7 @@ private void preEmulatorSetupCheck() {
 		sparkHistoryEndpoint = sparkHistoryEndpointField.getText().trim().replaceAll("/+$", "");
 		ambariEndpoint = ambariEndpointField.getText().trim().replaceAll("/+$", "");
 
+		AppInsightsClient.create(Messages.HDInsightCreateLocalEmulator, null);
```

Comment:
The `Messages` here not import. Seems the `Messages.HDInsightCreateLocalEmulator` not work here, it is belong to another module `com.microsoft.azuretools.hdinsght`.